packages/react-app/src/App.jsx

```diff
@@ -23,15 +23,15 @@ const DEBUG = true;
 const NETWORKCHECK = true;
 
 // Add more networks as the dapp expands to more networks
-const configuredNetworks = ["mainnet", "rinkeby", "xdai", "matic", "mainnetAvalanche"];
```

Comment:
Why remove the other networks? This is needed for the network switch

Reply:
Didn't deploy the new contract on those networks yet, if you try switching to those networks, it will cause an error.
We can add them back once we deploy the new contract to those networks.

Reply:
We're still testing, so just having rinkeby is fine

Reply:
It would be better to comment out the line, then we know networks the old contracts were deployed to

Reply:
Okay that's a good idea